src/devtools/client/debugger/src/components/Editor/ToggleWidgetButton.tsx

```diff
@@ -153,7 +150,13 @@ function QuickActions({
 
         // If hit counts are shown, the button should not overlap with the gutter.
         // The gutter size changes though based on the number of hits, so we use a CSS variable.
-        right: hitCounts ? "var(--hit-count-gutter-width)" : undefined,
+        right:
+          // we should move this util to a shared function
+          hitCountsMode === "show-counts"
+            ? "calc(var(--hit-count-gutter-width) - 6px)"
+            : hitCountsMode === "hide-counts"
+            ? "-10px"
+            : "0px",
```

Comment:
This whole thing looks super sketchy.

Reply:
Like any time I see hard-coded pixel values like this, I assume it will break for things like our large/small font toggle– or that it will be fragile to any other border/padding/margin changes we happen to make elsewhere.

Reply:
sure. would love your thoughts on how we can improve this too. The core problem is pretty simple. Basically teach the print statement button to be slightly to the right of the gutter